docs/getting-started.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Getting Started"
+date: 2020-08-26T10:56:03
+weight: 2
+geekdocRepo: https://github.com/owncloud/file-picker
+geekdocEditPath: edit/master/docs
+geekdocFilePath: getting-started.md
+---
+
+{{< toc >}}
+
+ownCloud File picker is a web component which can be integrated into existing web applications. It connects to ownCloud server and enables user to select resources which are then provided in a response of a fired event. Visit [instalation]({{< ref "instalation.md" >}}) to see how to integrate File picker into your product.
+
+## Components of the File picker
+File picker can be used in two different ways: File picker and location picker.
```

Comment:
"**The** file picker" sounds better ?

Also I'd recommend to use the term "variation" (not ways) which you used further down.